### ISSUE TITLE
[Screen Reader - Documentation] Narrator focus is navigating to hidden element in scan mode and reading it as 'blank'

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2801]
**User Experience:**
If Narrator in scan mode is navigating to hidden element and reading as blank then user with visual disabilities may get misguided to understand the behavior of the controls

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to url:-https://makecode.microbit.org/device/error-codes
2-Navigate to all element on error code page
3- launch narrator change to scan mode by (caps+ space key)
4-Verify if screen reader moving to any hidden element or not

**Actual Result:**
Narrator in Scan mode is navigating to hidden element and reading as blank

**Expected Result:**
In scan mode Narrator focus should not navigate to hidden element

**Note:**
In item mode narrator is navigating to hidden element and reading as BBC microbit simulator

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

Refer Attachment:
[MAS4.2.10_FocusmovingtohiddenElement.zip](https://github.com/Microsoft/pxt/files/1248667/MAS4.2.10_FocusmovingtohiddenElement.zip)

**Comment by veler** [comment]
This is linked to a bad behavior from blocky and has already been pointed several times in some previous issues.

**Comment by abchatra** [comment]
We have got exception for Blockly bugs from Julia Liuson (CVP). Closing these as Won't Fix. Following is the reasoning.

----------------------

MakeCode can be used in either Text editor or Block editor.

Blocks editor is inherently visual and is based on google blockly. Usage requires drag and drop functionality and we don’t have a equivalence in MakeCode for this w.r.t accessibility. 
We have evaluated couple of research projects which tries to find a equivalence to block based coding for accessibility needs: 
•	Project Torino (Requires students to buy physical blocks)
•	Accessible blockly (This is text based coding without syntax errors)
Both these are drastic departure from the blockly and provides a complete new experience. These are still in research and we intend not to use them in MakeCode at this point of time.

As workaround, the text editor in MakeCode is the way we see students with the need of accessibility using MakeCode.
